tests/speculos/conftest.py

```diff
@@ -14,7 +14,7 @@
 
```

Comment:
Ok I don't understand why we have two conftest in two directories?
We can run `pytest tests/speculos`
and `pytest tests/ragger`
Why not unify everything ?

Reply:
To go even further, I think Speculos tests and ledgercomm tests should be removed and replaced by only ragger tests? After all, that's the purpose of ragger :)

Reply:
I'll remove the older tests examples.